samples/nextjs/src/lib/page-props.ts

```diff
@@ -1,6 +1,9 @@
-import { DictionaryPhrases, LayoutServiceData } from '@sitecore-jss/sitecore-jss-nextjs';
+import {
+  DictionaryPhrases,
+  LayoutServiceData,
+  ComponentProps,
```

Comment:
Should we name this type something more specific to its dictionary/collection purpose, like "ComponentPropsCollection"? I could see it being confusing (vs those types in my "component-props.ts"). 

The property name "componentProps" on the SitecorePageProps type is fine I think.

Reply:
Agree, it looks more clear, when I use Collection word